ldes-server-compaction/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/compaction/application/services/CompactionScheduler.java

```diff
@@ -15,35 +17,43 @@
 
 @Service
 public class CompactionScheduler {
+	private static final Logger LOGGER = LoggerFactory.getLogger(CompactionScheduler.class);
 	private final ViewCollection viewCollection;
 	private final FragmentRepository fragmentRepository;
 	private final PaginationCompactionService paginationCompactionService;
+	private final CompactionCandidateService compactionCandidateService;
 
 	public CompactionScheduler(ViewCollection viewCollection, FragmentRepository fragmentRepository,
-			PaginationCompactionService paginationCompactionService) {
+	                           PaginationCompactionService paginationCompactionService,
+	                           CompactionCandidateService compactionCandidateService) {
 		this.viewCollection = viewCollection;
 		this.fragmentRepository = fragmentRepository;
 		this.paginationCompactionService = paginationCompactionService;
-
+		this.compactionCandidateService = compactionCandidateService;
 	}
 
 	@Scheduled(cron = COMPACTION_CRON_KEY)
 	public void compactFragments() {
 		viewCollection.getAllViewCapacities()
 				.parallelStream()
-				.forEach(viewCapacity -> getRootFragment(viewCapacity)
-						.ifPresent(rootFragment -> {
-							PaginationStartingNodeIterator paginationStartingNodeIterator = new PaginationStartingNodeIteratorImpl(
-									fragmentRepository, rootFragment);
-							while (paginationStartingNodeIterator.hasNext()) {
-								Fragment next = paginationStartingNodeIterator.next();
-								paginationCompactionService
-										.applyCompactionStartingFromNode(next);
-							}
-						}));
+				.forEach(viewCapacity -> getRootFragment(viewCapacity).ifPresent(rootFragment -> {
+
+					var compactionTaskList = compactionCandidateService.getCompactionTaskList(viewCapacity);
+
+					if (compactionTaskList.isEmpty()) {
+						LOGGER.info("No compaction candidates available for " + viewCapacity.getViewName().getViewName());
+					}
+					else {
```

Comment:
Can you put this one line up to be } else { , to have formatting in line with everywhere else in the application.